Description: Fixes the clean target of Makefile. Removes no longer necessary
 after installation directory python/build.

Author: Kęstutis Biliūnas <kebil@kaunas.init.lt>
---
Forwarded: Peter Baker <psb6m@virginia.edu> 
Last-Update: 2010-06-17

--- xgridfit-2.2a.orig/Makefile
+++ xgridfit-2.2a/Makefile
@@ -137,6 +137,7 @@ clean:
 	rm -f schemas/*.*~
 	rm -f utils/*.*~
 	rm -f man/*.*~
+	rm -fr python/build
 
 
 .PHONY:	install install-mac install-docs install-all install-all-mac \
